pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Thank you for your Pull Request. Please provide a description and review
+the requirements below.
+
+Contributors guide: https://github.com/eclipse-jdt/.github/blob/main/CONTRIBUTING.md
+
+Note: Security vulnerabilities should not be disclosed on GitHub, through a PR or any
+other means. TODO: how to report security issues
+-->
+
+## What it does
+<!-- Include relevant issues and describe how they are addressed. -->
+
+## How to test
+<!-- Explain how a reviewer can reproduce a bug, test new functionality or verify performance improvements. -->
+
+## Author checklist
+
+- I have thoroughly tested my changes
+- The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
+- I have signed the the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
```

Comment:
Minor typo (too many "the"s).  Is the ECA not enforced by github for Eclipse in the checks?

Reply:
Fixed.